fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/query/GroupByTest.java

```diff
@@ -237,33 +257,220 @@ private Reference constructGroupByPlan(final boolean withPredicateInSelectWhere,
         }
     }
 
-    protected void setupHookAndAddData(final boolean addIndex, final boolean addAggregateIndex) {
+    @DualPlannerTest(planner = DualPlannerTest.Planner.CASCADES)
+    void testBitMapWithStreamAggregation() {
```

Comment:
for this, could only work with queries like `select a, bitmap(b % 100) from Foo group by a`. it doesn't work with `group by bit_bucket(b, 100)`.

Reply:
I don't think I understand this comment.